rbd-target-api.py

```diff
@@ -1008,7 +1009,15 @@ def disk(pool, image):
     if request.method == 'GET':
 
         if image_id in config.config['disks']:
-            return jsonify(config.config["disks"][image_id]), 200
+            disk_dict = config.config["disks"][image_id]
+            global dev_status_watcher
+            disk_status = dev_status_watcher.get_dev_status(image_id)
+            if disk_status:
+                disk_dict['status'] = disk_status.get_status_dict()
+            else:
+                disk_dict['status'] = 'Unknown'
```

Comment:
Nit: should this be assigning a dict to keep it consistent: 
```suggestion
                disk_dict['status'] = {'state': 'Unknown'}
```

Reply:
Sure.